src/llmcompressor/modifiers/quantization/quantization/base.py

```diff
@@ -10,17 +10,19 @@
     is_preset_scheme,
     preset_name_to_scheme,
     set_module_for_calibration,
+    ActivationOrdering,
 )
 from compressed_tensors.quantization.observers.helpers import get_observer_token_count
 from loguru import logger
-from pydantic import Field
+from pydantic import Field, field_validator
 from torch.nn import Module
 
 from llmcompressor.core import Event, EventType, State
 from llmcompressor.modifiers import Modifier
 from llmcompressor.modifiers.utils.pytorch_helpers import run_calibration_forward
 
-__all__ = ["QuantizationModifier"]
+__all__ = ["QuantizationModifier", "CONTIGUOUS_ACTIVATION_ORDERINGS"]
+CONTIGUOUS_ACTIVATION_ORDERINGS = {ActivationOrdering.GROUP}
```

Comment:
isn't `ActivationOrdering.GROUP` the only non-contiguous one? not the only contiguous one?  `CONTIGUOUS_ACTIVATION_ORDERINGS`

Reply:
Yeah sorry I inverted the set accidentally